src/main/java/school/lemon/changerequest/java/cafe/domain/Client.java

```diff
@@ -9,15 +11,16 @@
 
     private final Random random = new Random();
 
-    public void drinkCoffee(Drink drink) {
+    public void drinkCoffee(Drink drink) throws IncorrectDrinkException, LowTemperatureException,
+            HighTemperatureException, RandomClientException {
         if (drink.getDrinkType() != DrinkType.COFFEE) {
-            System.out.println("Tastes like a " + drink.getDrinkType());
+            throw new IncorrectDrinkException("Tastes like a " + drink.getDrinkType());
         } else if (drink.getTemperature() < LOWEST_TEMPERATURE) {
-            System.out.println("Looks like it's too cold...");
+            throw new LowTemperatureException("Looks like it's too cold...", drink.getTemperature());
         } else if (drink.getTemperature() > HIGHEST_TEMPERATURE) {
-            System.out.println("Looks like it's too hot...");
+            throw new HighTemperatureException("Looks like it's too hot...", drink.getTemperature());
         } else if (random.nextInt(100) < 10) {
-            System.out.println("I don't like your Cafe!");
+            throw new RandomClientException("I don't like your Cafe!");
```

Comment:
It's definitely not `random` exception. It's an exception that says smth like: 'Hey, I don't like you cafe' — maybe smth is wrong with serving or anything else, but please create meaningful exception name.